src/tools/OptionsSchema.json

```diff
@@ -0,0 +1,398 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Sounds reasonable. Note that VS Code generally allows comments in all json files, but I am not sure how easy it is to make other json readers ignore comments...